src/Components/test/E2ETest/ServerExecutionTests/ComponentHubInvalidEventTest.cs

```diff
@@ -71,7 +71,7 @@ public async Task DispatchingAnInvalidEvent_DoesNotTriggerWarnings()
             {
                 BrowserRendererId = 0,
                 EventHandlerId = 1990,
-                EventArgsType = "mouse",
+                EventName = "click",
```

Comment:
I get the reason for the property name change, but why are we changing the value here?

Reply:
Because `mouse` was the name of the category (which included click events), and we're no longer supplying category names. We're now supplying the actual event name, which in this case is `click`.